samples/job_definitions/model_explainability/README.md

```diff
@@ -0,0 +1,57 @@
+# Model Explainability Job Definition Sample
+
+This sample demonstrates how to start model-explainability job definitions using your own model-explainability script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do).
+
+You will also need an [Endpoint](/samples/endpoint/README.md) configured in SageMaker and will also need to run baselining jobs using the Python SageMaker SDK before you can use it with ACK.
```

Comment:
~~will also need to run baselining jobs using the Python SageMaker SDK before you can use it with ACK~~

you will need to run a baselining job to generate baseline constraints.